torch/csrc/jit/ir.cpp

```diff
@@ -170,8 +174,21 @@ void printAttributes(std::ostream & out, Node * n) {
         printPrimList(out,n->ss(name));
         break;
       case AttributeKind::t:
-        out << "<Tensor>";
-        break;
+        {
+          at::Tensor t = n->t(name);
+          if (t.numel() <= max_tensor_display_size) {
+            // TODO: This is awful code.  Also it doesn't work on Windows.
```

Comment:
Why doesn't it work on windows?

Reply:
Windows newlines are \r\n, but this will only search replace \n. But going from one character to multi-character search replace requires exercising different C++ stuff